src/js/components/container/ButtonContainer.js

```diff
@@ -1,22 +1,53 @@
 import React, { Component } from "react";
```

Comment:
Can you change the name of the class from ButtonContainer.js to something more indicative of what the class/button is?

Reply:
Done!